packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:html' as html;
+import 'dart:html';
```

Comment:
Do we need this 2nd import? I think you can just reference CanvasElement w/ a html. prefix.

Reply:
nope. fixed